apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -75,15 +76,25 @@ exports.ApplicationFinalStatuses = [
  * Constants for Regional Partner dropdown
  */
 const allPartnersLabel = "All Regional Partners' Applications";
-const allPartnersFilter = "all";
-const unmatchedLabel = "No Partner/Unmatched";
-const unmatchedFilter = "none";
+const allPartnersValue = "all";
+const unmatchedPartnerLabel = "No Partner/Unmatched";
+const unmatchedPartnerValue = "none";
 exports.AllPartnersLabel = allPartnersLabel;
-exports.AllPartnersFilter = allPartnersFilter;
-exports.UnmatchedLabel = unmatchedLabel;
-exports.UnmatchedFilter = unmatchedFilter;
+exports.AllPartnersValue = allPartnersValue;
+exports.UnmatchedPartnerLabel = unmatchedPartnerLabel;
+exports.UnmatchedPartnerValue = unmatchedPartnerValue;
```

Comment:
nit: I know this has come up before, but this seems unwieldy. Unless we have a compelling reason to duplicate all these constants, how about defining and exporting at once like this:
```es6
export const AllPartnersLabel = "All Regional Partners' Applications";
export const AllPartnersValue = "all";
// etc
```

Reply:
After some discussion, and consulting our styleguide, we should label them uppercase, e.g.
```es6
export const ALL_PARTNERS_VALUE = "all";
```

Reply:
i'll follow up with a PR fixing these and other constants